bundle/config/presets.go

```diff
@@ -17,6 +17,11 @@ type Presets struct {
 	// JobsMaxConcurrentRuns is the default value for the max concurrent runs of jobs.
 	JobsMaxConcurrentRuns int `json:"jobs_max_concurrent_runs,omitempty"`
 
+	// InPlaceDeployment indicates whether in-place deployment is enabled. Works only in workspace
+	// When set to true, resources created during deployment will point to source files in the workspace instead of their workspace copies.
+	// No resources will be uploaded to workspace
+	InPlaceDeployment *bool `json:"in_place_deployment,omitempty"`
```

Comment:
The naming is a bit confusion tbh, I'd better use something like `NoFilesUpload` or `UseWorkspaceFiles` or smth.